CMakeLists.txt

```diff
@@ -140,6 +140,30 @@ else()
   endif()
 endif()
 
+# Add new build types
+message("* Adding coverage build type...")
+SET(CMAKE_CXX_FLAGS_COVERAGE
```

Comment:
Instead of adding a new build-type, please enable code-coverage by passing in `-fprofile-arcs -ftest-coverage` as a flag to cmake in travis.sh.

Reply:
Done